build.gradle.kts

```diff
@@ -61,7 +61,7 @@ allprojects {
     apply(plugin = "distribution")
 
     multiJvm {
-        jvmVersionForCompilation.set(11)
+        jvmVersionForCompilation.set(17)
```

Comment:
We have already discussed about a month ago why we need Java 17: GraphQL Kotlin needed to support SpringBoot v3 which requires a Java version >= 17. From the last time we talked about this, you said it would not be problematic.
If you prefer I can keep the Java version to 11 in the root build file, and specify version 17 for my sub-project only.

Reply:
Nope, it is fine. We raise the requirement.